### DISCUSSION TITLE
A few lists of wordlists

### DISCUSSION THREAD

**Discussion by Xzar-x** [epi052/feroxbuster#1195] [Q&A]
Hello can u make an option to add a few different wordlists ? Something like in ffuf e.g. "ffuf -u http://94.237.60.204:38628/webfuzzing_hidden_path/flag/FUZZEXT -w /usr/share/seclists/Discovery/Web-Content/big.txt:FUZZ -w /usr/share/seclists/Discovery/Web-Content/raft-small-extensions.txt:EXT" => I scan a file and extensions at once. And i have got one more request could you add possibility scan DNS or VHOST ?? I love use feroxbuster but sometimes i have to use alternately ffuf.

**Comment by tipilegal** [comment]
For your first use case you actually do not need two wordlists. feroxbuster handles extensions natively with `-x`, and it is flexible about how you pass them:

```
feroxbuster -u http://94.237.60.204:38628/webfuzzing_hidden_path/flag/ -w /usr/share/seclists/Discovery/Web-Content/big.txt -x txt,php,html
```

That appends each extension to every word in the wordlist, which gives you the same result as your `FUZZ` + `EXT` example in ffuf. The styles are interchangeable and get merged, so `-x pdf -x js,html -x php txt` works too.

What is genuinely not supported is the general case: multiple wordlists bound to named keywords, pitchfork/clusterbomb style. That is tracked in #242, still open, so for anything beyond path+extension you will keep reaching for ffuf.

On VHOST/DNS, that is out of scope for this tool. feroxbuster does recursive content discovery, it fuzzes paths under a host you already know rather than fuzzing the host itself. For those use `ffuf -H "Host: FUZZ.target.com"` or gobuster in `vhost` / `dns` mode.
